data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -12,6 +12,17 @@ function wait_for_api {
   done
 }
 
+# This is required since the progress service (https://github.com/openshift/installer/blob/master/data/data/bootstrap/files/usr/local/bin/report-progress.sh#L22-L33),
+# usually dedicated to create the bootstrap ConfigMap, will fail in case of SingleNodeOpenShift deployment due the lack of a control plane
```

Comment:
```suggestion
# usually dedicated to creating the bootstrap ConfigMap, will fail to create this configmap in case of bootstrap-in-place single node deployment, due to the lack of a control plane when bootkube is complete
```

Reply:
(The pedantic edit is because technically a SNO installatoin with a bootstrap node will not have this problem. This problem only affects bootstrap-in-place)